app/models/competition.rb

```diff
@@ -298,10 +298,6 @@ def number_of_events
     persisted_events_id.length
   end
 
-  def has_administrative_notes?
-    registrations.any? { |registration| registration.administrative_notes.present? }
-  end
-
```

Comment:
This method is okay to remove because it would otherwise conflict with the new cops